### ISSUE TITLE
API issue 

### ISSUE THREAD

**Issue by nagkumar** [CodeIntelligenceTesting/jazzer#753]
Trying this example  https://www.code-intelligence.com/blog/fuzz-java-cifuzz-cli?utm_source=pocket_saves

```
> Task :compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':runtimeClasspath'.
   > Could not find com.code-intelligence:jazzer-api:0.17.0.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/code-intelligence/jazzer-api/0.17.0/jazzer-api-0.17.0.pom
     If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
     Required by:
         project :
         project : > com.code-intelligence:jazzer-junit:0.17.0
         project : > com.code-intelligence:jazzer-junit:0.17.0 > com.code-intelligence:jazzer:0.17.0

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

while the API jar is at https://repo1.maven.org/maven2/com/code-intelligence/jazzer-api/0.17.0/

**Comment by bertschneider** [comment]
Thanks for the heads up. We are looking into the latest maven release and, as you said, something clearly went wrong there.

**Comment by fmeum** [comment]
We just released v0.17.1 as a replacement for v0.17.0, which got corrupted before reaching Maven Central. Please try again with the fixed release and reopen if this keeps showing up.

**Comment by nagkumar** [comment]
Thank You for the speedy fix,  tested it with v0.17.1  it works as expected..
